images/macos/provision/core/powershell.sh

```diff
@@ -31,7 +25,4 @@ pwsh -command "& {Import-Module Az}"
 # powershell link was removed in powershell-6.0.0-beta9
 sudo ln -s /usr/local/bin/pwsh /usr/local/bin/powershell
 
-# fix ~/.azure directory permissions
-sudo chown -R ${USER}: $HOME/.azure
```

Comment:
Looks like we drop this line? Are you sure that it is safe and won't break users?

Reply:
@maxim-lobanov Yes, we are ok here, I tested it and the directory and the files in it are created with the standard user's permissions and ownership, might be the fix was needed before, but now it is ok per se.